Source/WTF/WTF.xcodeproj/project.pbxproj

```diff
@@ -109,6 +109,7 @@
 		70A993FE1AD7151300FA615B /* SymbolRegistry.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 70A993FC1AD7151300FA615B /* SymbolRegistry.cpp */; };
 		70ECA60D1B02426800449739 /* AtomStringImpl.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 70ECA60A1B02426800449739 /* AtomStringImpl.cpp */; };
 		7A05093F1FB9DCC500B33FB8 /* JSONValues.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 7A05093E1FB9DCC500B33FB8 /* JSONValues.cpp */; };
+		7A4228942ABE0FF60065F7B5 /* PlatformEnableGTKWPE.h in Headers */ = {isa = PBXBuildFile; fileRef = 7A4228932ABE0FF60065F7B5 /* PlatformEnableGTKWPE.h */; };
```

Comment:
It'd be nice to use "Glib" instead of "GTKWPE". "glib" is already the abbreviation used in layout tests to mean both GTK and WPE.

Reply:
That's a great idea -- I'll make that change and run through EWS.